pkg/service/utgen/assets/test_generation.toml

```diff
@@ -180,6 +267,9 @@ Use block scalar('|') to format each YAML output.
 {% endif %}
 
 
+In Python, since this can be an issue Unclosed client session client_session: <aiohttp.client.ClientSession object at 0x102fed130>, please ensure using pytest-aiohttp to solve this issue
```

Comment:
I don't think this is correct

Reply:
Tests are failing due to unclosed session, I isolated it

Reply:
we can do a better thing, please check if we send this as error, print the prompt probably it should have had the error (this seems bit focused with address)